src/vs/workbench/contrib/positronNotebook/browser/PositronNotebookInstance.ts

```diff
@@ -962,11 +961,6 @@ export class PositronNotebookInstance extends Disposable implements IPositronNot
 	 * @param cell The cell to clear outputs from. If not provided, uses the currently selected cell.
 	 * @param skipContentEvent If true, won't fire the content change event (useful for batch operations)
 	 */
-	/**
-	 * Clears the output of a specific cell in the notebook.
-	 * @param cell The cell to clear outputs from. If not provided, uses the currently selected cell
-	 * @param skipContentEvent If true, won't fire the content change event (useful for batch operations)
-	 */
```

Comment:
I did some nasty rebases that caused all sorts of these duplicates. Thanks for catching!